code/game/machinery/portable_turret.dm

```diff
@@ -438,6 +441,7 @@
 		settings[++settings.len] = list("category" = "Check Access Authorization", "setting" = "check_access", "value" = check_access)
 		settings[++settings.len] = list("category" = "Check misc. Lifeforms", "setting" = "check_anomalies", "value" = check_anomalies)
 		settings[++settings.len] = list("category" = "Neutralize All Entities", "setting" = "check_all", "value" = check_all)
+		settings[++settings.len] = list("category" = "Neutralize Downed Entities on Lethal", "setting" = "check_down", "value" = check_down)

```

Comment:
maybe "Eliminate Downed Entities" instead- just because "Neutralize Downed Entities on Lethal" sounds too clunky

Reply:
That'd likely lead to confusion as it wouldn't shoot downed people when the lethal setting is not enabled.